File: samples/CSharp/Northwind/EmployeeTerritory.cs
using System;
using DbExtensions;

namespace Samples.CSharp.Northwind {

   [Table(Name = "EmployeeTerritories")]
   public class EmployeeTerritory {

      [Column(IsPrimaryKey = true)]
      public int EmployeeID { get; set; }

      [Column(IsPrimaryKey = true)]
      public string TerritoryID { get; set; }

      [Association(ThisKey = nameof(EmployeeID))]
      public Employee Employee { get; set; }

      [Association(ThisKey = nameof(TerritoryID))]
      public Territory Territory { get; set; }
   }
}

File: samples/VisualBasic/Northwind/EmployeeTerritory.vb
Imports System
Imports DbExtensions

Namespace Northwind

   <Table(Name:="EmployeeTerritories")>
   Public Class EmployeeTerritory

      <Column(IsPrimaryKey:=True)>
      Property EmployeeID As Integer

      <Column(IsPrimaryKey:=True)>
      Property TerritoryID As String

      <Association(ThisKey:=NameOf(EmployeeID))>
      Property Employee As Employee

      <Association(ThisKey:=NameOf(TerritoryID))>
      Property Territory As Territory

   End Class

End Namespace
